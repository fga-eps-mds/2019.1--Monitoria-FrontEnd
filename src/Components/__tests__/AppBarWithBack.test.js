import React from 'react';
import AppBarWithBack from '../Feed/AppBarWithBack';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { MemoryRouter as Router } from 'react-router-dom';

const appbarwithback = jest.fn();

  it('Test if AppBarrWithBack renders correctly', () =>{
    const tree = renderer.create(
      <Router><AppBarWithBack profile={appbarwithback}/></Router>
      )
      expect(toJson(tree)).toMatchSnapshot()
  });