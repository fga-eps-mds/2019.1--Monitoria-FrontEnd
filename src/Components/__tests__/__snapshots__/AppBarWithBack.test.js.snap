// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if AppBarrWithBack renders correctly 1`] = `undefined`;

exports[`Testing AppBarWithBack component Test if AppBarWithBack renders correctly 1`] = `
<SimpleAppBar
  classes={
    Object {
      "root": "SimpleAppBar-root-1",
    }
  }
/>
`;
